server/autolinkplugin/command.go

```diff
@@ -23,6 +23,7 @@ const (
 	optDisableNonWordPrefix = "DisableNonWordPrefix"
 	optDisableNonWordSuffix = "DisableNonWordSuffix"
 	optWordMatch            = "WordMatch"
+	newLine                 = "```n"
```

Comment:
This is missing the backslash. Can you make sure the change works correctly? Alternatively, we can disable this linting check. I think I would prefer that, because this is a clear case where it's working against us and not providing much value

Also, there is more going on here than a "new line". We're also closing a code block